_data/learnnavswanlake.yml

```diff
@@ -13,6 +13,9 @@
     - title: Writing Your First Ballerina Program 
       url: /learn/user-guide/getting-started/writing-your-first-ballerina-program/
       active: writing-your-first-ballerina-program
+    - title: Writing a gRPC service with Ballerina
```

Comment:
```suggestion
    - title: Writing a gRPC Service with Ballerina
```

Same here.

Reply:
Please verify if all titless are in title case.

Reply:
We need to define all the sub-topics also in this YML file.

Reply:
There are no sub-tops in this guide. Only main title is enough IMO